systemd/osrt-totest-manager@.service

```diff
@@ -2,11 +2,8 @@
 Description=openSUSE Release Tools: ToTest Manager for %i
 
 [Service]
+Type=oneshot
```

Comment:
I 100% agree with this based on documentation, but @lnussel ruled to use default (ie simple). So lets be consistent at the least. :)